lib/index.js

```diff
@@ -2448,6 +2448,8 @@ exposeMethod('split');
 
 /**
  * Creates a new Stream with the values from the source in the range of `start` to `end`.
+ * `start` and `end` must be of type `Number`, if not the whole stream will
```

Comment:
Rather, `start` defaults to `0` if not a number and `end` defaults to `Infinity`. The way it's worded right now makes it seem like `s.slice(3, "a string")` would return the whole stream.